soda_machine.rb

```diff
@@ -1,18 +1,29 @@
+require_relative 'soda'
+
 class SodaMachine
-  attr_reader :sodas, :cash
+  attr_reader :sodas, :cash, :inventory
 
   def initialize(args = {})
     @sodas = args[:sodas]
     @cash = args[:cash]
+    @inventory = []
```

Comment:
No need to add a new instance variable -- `@sodas` already contains the inventory of sodas that the machine currently has in it.